src/reactions/Reaction.js

```diff
@@ -26,24 +28,79 @@ const styles = StyleSheet.create({
   frameNotVoted: {
     borderColor: '#dedede', // eslint-disable-line
   },
-  countCommon: {
+  placeHolderCount: {
     marginLeft: 4,
+    color: 'transparent'
   },
   countVoted: {
     color: BRAND_COLOR,
   },
   countNotVoted: {
     color: 'gray',
   },
+  roller: {
+    position: 'absolute',
+    top: -20,
+    right: 4,
+  },
+  rollerText: {
+    paddingTop: 3,
+    paddingBottom: 3
+  }
 });
 
+const forwardAnimationConfig = {
+  fromValue: 0,
+  toValue: 1,
+  duration: 400,
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+};
+
+const backAnimationConfig = {
+  ...forwardAnimationConfig,
+  fromValue: 1,
+  toValue: 2
+};
+
 class Reaction extends React.PureComponent {
 
+  state = {
+    voteChangeAnimation: new Animated.Value(0)
+  };
+
   props: {
     name: string,
     voted: boolean,
   };
 
+  componentWillReceiveProps = (nextProps) => {
+    const { voteCount } = this.props;
+
+    if (nextProps.voteCount > voteCount) {
+      this.incrementCounter();
```

Comment:
I created a separate method anticipating that it may house more logic to prevent false updates if the button is tapped repeatedly. If it remains a one liner definitely inlining is the way to go.

Also, even if it remains a single line method, keeping readability in mind don't you think using `this.incrementCounter();` is slightly better than `this.animation(Animated.timing, 'voteChangeAnimation', forwardAnimationConfig).start();` ?